tldraw/tldraw
# feat: add Somali language localization

This pull request adds Somali language localization to the tldraw application, enabling users to view the interface in Somali. The new translation covers all UI elements, menus, tooltips, and actions, improving accessibility for Somali-speaking users.

### Change type


### Test plan

1. Set the application language to Somali and verify that all interface elements display in Somali correctly.
2. Confirm that the layout and functionality remain consistent with other languages.


### Release notes

- Added Somali language localization to the tldraw interface
